### PR TITLE
[OSDOCS6507] Restructure bare metal config parms to book level

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-6507

Link to docs preview:

- [Installation configuration parameters for bare metal](https://63621--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installation-config-parameters-bare-metal.html)
New assembly at book level
- Installing a cluster on bare metal [Manually creating the installation configuration file](https://63621--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-initializing-manual_installing-bare-metal)
Note the absence of Installation configuration parameters following this topic. In liue of the module, a cross reference is provided for the new assembly.
It is worth noting that the latter is an example of the change that this PR introduces. This change has been applied to all bare metal UPI installation assemblies.

QE review:
- [NA ] QE has approved this change.
  Given that this update is a reorg only, QE approval is not required. See conversation here: https://github.com/openshift/openshift-docs/pull/61165#issuecomment-1600861041. 

Additional information:

### CODE REVIEW

**Comment by SNiemann15** [comment]
@mjpytlak Only `installation-configuration-parameters.adoc` showed conflicts that I had to resolve. I checked the ASH files in my preview and your changes got in. 
Only thing I noticed is that Installing a cluster on Azure Stack Hub using ARM templates doesn't have an additional resources link to the new installation configuration parms assembly. Is that on purpose? 

https://63621--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra#installation-initializing-manual_installing-azure-stack-hub-user-infra

**Comment by mjpytlak** [comment]
> Only thing I noticed is that **Installing a cluster on Azure Stack Hub using ARM templates** doesn't have an additional resources link to the new installation configuration parms assembly. Is that on purpose

@SNiemann15 Thanks for checking. Yes, that is by design. That assembly never used `installation-configuration-parameters.adoc` and the procedure you reference is prescriptive in what needs updating.

**Comment by SNiemann15** [comment]
/label peer-review-needed

@sjstout next one :-) PTAL

**Review by jeana-redhat — COMMENTED** [review]
One Q, otherwise LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc
[the remark this answers is not in the document]
Comment (SNiemann15) [reply installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc:1]: Yes that is intended. Cross reference was missing see discussion above. Additional resource was missing here: https://63621--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra#installation-initializing-manual_installing-azure-stack-hub-user-infra

**Comment by sjstout** [comment]
LGTM!

**Comment by mjpytlak** [comment]
Nice work @SNiemann15.

**Pull request merged**